pandas/tests/frame/test_analytics.py

```diff
@@ -1201,6 +1201,13 @@ def test_isin_df(self):
         expected['B'] = False
         tm.assert_frame_equal(result, expected)
 
+    def test_isin_tuples(self):
+        df = pd.DataFrame({'A': [1, 2, 3], 'B': ['a', 'b', 'f']})
+        df['C'] = list(zip(df['A'], df['B']))
+        result = df['C'].isin([(1, 'a')])
+        tm.assert_series_equal(result,
+                              Series([True,False,False],name="C"))
```

Comment:
for PEP8: the indentation does not fully match with the line before (one space more needed), and a space after the comma is needed